samples/cpp/benchmark_app/main.cpp

```diff
@@ -1253,8 +1253,11 @@ int main(int argc, char* argv[]) {
             statistics->dump();
 
         // Performance metrics report
-        if (device_name.find("AUTO") != std::string::npos)
-            slog::info << "ExecutionDevice:   " << compiledModel.get_property(ov::execution_devices) << slog::endl;
+        try {
+            auto exeDevice = compiledModel.get_property(ov::execution_devices);
+            slog::info << "ExecutionDevice:   " << exeDevice << slog::endl;
+        } catch (...) {
```

Comment:
Maybe catch specific exception just to skip non implemented only?

Reply:
fixed